torch/nn/_functions/rnn.py

```diff
@@ -343,10 +343,29 @@ def backward_extended(self, grad_output, grad_hy):
 
 
 def RNN_symbolic_builder(*args, **kwargs):
-    def symbolic(g, input, all_weights, hx, **kwargs):
-        # Something can go here, e.g.
-        # return g.op('LSTM', input, *all_weights[0], outputs=2)
-        raise RuntimeError("RNN symbolic NYI")
+    assert args[0] == 'LSTM'
+
+    num_layers = kwargs['num_layers']
+    hidden_size = args[2]
+
+    def symbolic(g, input, all_weights, h0_and_c0, **fkwargs):
+        h0, c0 = h0_and_c0
+        sequence_len = input.type().sizes()[0]
+        batch_size = input.type().sizes()[1]
+
+        weight_ih = all_weights[0][0]
+        weight_hh = all_weights[0][1]
+        bias_concat = g.op('Concat', all_weights[0][2], all_weights[0][3], axis_i=0)
+        sequence_lens = g.op('Constant', value_t=torch.IntTensor(batch_size).fill_(sequence_len))
```

Comment:
this may have to be a future extension - the problem is that I need to provide following optional arguments, and just leaving this out or proving None instead doesn't currently work